src/api/app/controllers/request_controller.rb

```diff
@@ -74,10 +72,7 @@ def global_command
   def request_command
     return request_command_diff if params[:cmd] == 'diff'
 
-    # refuse request manipulation for anonymous users
-    require_login
-
-    params[:user] = User.session!.login
+    params[:user] = User.session.login
```

Comment:
How can we ensure `User.session` is filled if we do not `require_login`?

Reply:
We already do `require_login`. `RequestController` inherits from `ApplicationController`, and `require_login` is called in `ApplicationController`.

Reply:
Alright